src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/__tests__/HealthCertificateValidationServiceValidationTests.swift

```diff
@@ -0,0 +1,13 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+@testable import ENA
+import HealthCertificateToolkit
+import XCTest
+import SwiftCBOR
+import CertLogic
+
+class HealthCertificateValidationServiceValidationTests: XCTestCase {
```

Comment:
Is this needed? It is an empty class

Reply:
Yeah stub for the unit tests in a separate PR...